src/3rd_party_add.c

```diff
@@ -94,18 +94,12 @@ static int remove_repo(const char *repo_name)
 
 static bool confirm_certificate(const char *cert)
 {
-	int c;
-
 	info("Importing 3rd-party repository public certificate:\n\n");
 	signature_print_info(cert);
 
 	info("\n");
-	info("Do you want to accept this certificate? (y/N): ");
-
-	c = tolower(getchar());
-	info("\n");
-
-	return c == 'y';
+	info("To add the 3rd-party repository you need to accept this certificate. ");
```

Comment:
I would add a \n

Reply:
haha that is just what I was thinking... the line looked to long without it